client/src/app/components/create-member/create-member.component.ts

```diff
@@ -3,7 +3,7 @@ import { AuthService } from 'src/app/services/auth.service';
 import { FormGroup, FormBuilder, Validators } from '@angular/forms';
 import { Router } from '@angular/router';
 import { TokenService } from 'src/app/services/token.service';
-import roles from 'src/app/fixtures/roles.json';
+import roles from 'src/app/fixtures/create_members_roles.json';
```

Comment:
I think it is better to change the name of the import to be create_member_roles:

`import create_members_roles from 'src/app/fixtures/create_members_roles.json';`

and to rename the `roles` variable as `create_members_roles` in the component and html files so that it doesn't get mixed up with the `roles` variable used in other components.